src/plugins/intel_gpu/src/kernel_selector/core/cl_kernels/dft_ref.cl

```diff
@@ -0,0 +1,139 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#define GET_INDEX(prefix, ORDER) CAT(prefix, _GET_INDEX)(ORDER)
+
+// alternative: https://github.com/OpenCL/ComplexMath/blob/master/clcomplex.h
+typedef float2 cfloat;
```

Comment:
Please do not redefine this here, Because the cl codes are combined altogether and redefining here might cause error in the succeeding kernels expectation.